src/testUtils/TestRig.react.js

```diff
@@ -54,6 +54,11 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
+  toggleCheckbox(selector) {
+    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
```

Comment:
@maneeshnycda Per our conversation this should be disregarded: the efficiency of wrapping `element` only once isn't worth breaking the pattern used in every other function.